src/App.tsx

```diff
@@ -1,8 +1,110 @@
+import React, { useState } from 'react';
 import './App.css';
 
 function App() {
```

Comment:
Can you change function declaration to function expression? **e.g:** const App = () ...